tracee-rules/signature.go

```diff
@@ -0,0 +1,62 @@
+package main
+
+import (
+	"fmt"
+	"io/ioutil"
+	"log"
+	"os"
+	"path/filepath"
+	"plugin"
+
+	"github.com/aquasecurity/tracee/tracee-rules/types"
+)
+
+func getSignatures(rulesDir string, rules []string) ([]types.Signature, error) {
+	if rulesDir == "" {
+		exePath, err := os.Executable()
+		if err != nil {
+			log.Print(err)
+		}
+		rulesDir = filepath.Join(filepath.Dir(exePath), "rules")
```

Comment:
`signatures/golang/example.go` is not a usable signature, it's the source code for a couple of signatures. this code section needs compiled signatures which it looks for in this `rules` directory (as Amir asked). The rules directory is not part of the source code at all, it's expected to be found on the user's machine, and it's just a default value, in practice it can be any dir.